doc/nrf/ug_zigbee_qsg.rst

```diff
@@ -199,6 +203,17 @@ To program the samples, complete the following steps:
    On the development kit programmed with the light bulb sample, **LED 3** turns on when the light bulb joins the network.
    On the development kit programmed with the light switch sample, **LED 3** turns on when the device joins the network and **LED 4** turns on when the light switch finds a light bulb to control.
 
+.. note::
```

Comment:
I would add this as an indented header, that is something like this:
```suggestion
Reopening the network

   If you find that...
```
And maybe add "(see below)" at the end of line 193 ("Zigbee network needs to be reopened").

Not that I'm not a fan of notes, but in this case, I find it a bit long and maybe a pseudo-subsection would be better.